20:53:30  [INFO ] : {"status":200,"msg":"success","token":"6cfd5d003c7d463db2d70d919505e2a3"}
20:53:30  [INFO ] : 解析$.token得到的结果是6cfd5d003c7d463db2d70d919505e2a3存储为token
20:53:30  [INFO ] : {"status":200,"msg":"恭喜您，登录成功","userid":"10635"}
20:53:30  [INFO ] : 测试成功
20:53:30  [INFO ] : {"status":200,"msg":"用户已退出登录"}
20:53:30  [INFO ] : {"status":405,"msg":"非法请求"}
20:53:30  [INFO ] : 测试成功
20:53:31  [INFO ] : {"status":200,"msg":"success","token":"16a0269472c749e88a22e07be7af33b8"}
20:53:31  [INFO ] : 解析$.token得到的结果是16a0269472c749e88a22e07be7af33b8存储为token
20:53:31  [INFO ] : {"status":200,"msg":"恭喜您，登录成功","userid":"10635"}
20:53:31  [INFO ] : 测试成功
20:53:32  [INFO ] : {"status":200,"msg":"用户已退出登录"}
20:53:32  [INFO ] : {"status":200,"msg":"success","token":"8ff2dd6eb8ef43298ce34b1411a4f57b"}
20:53:32  [INFO ] : 解析$.token得到的结果是8ff2dd6eb8ef43298ce34b1411a4f57b存储为token
20:53:32  [INFO ] : {"status":402,"msg":"参数错误"}
20:53:32  [INFO ] : 测试成功
20:53:32  [INFO ] : {"status":200,"msg":"用户已退出登录"}
20:53:32  [INFO ] : {"status":200,"msg":"success","token":"f5839ed23de9406ca341aca410549444"}
20:53:32  [INFO ] : 解析$.token得到的结果是f5839ed23de9406ca341aca410549444存储为token
20:53:32  [INFO ] : {"status":402,"msg":"参数错误"}
20:53:32  [INFO ] : 测试成功
20:53:32  [INFO ] : {"status":200,"msg":"用户已退出登录"}
20:53:32  [INFO ] : {"status":200,"msg":"success","token":"417475e64ca64b0e9c5d16c94d0c9000"}
20:53:32  [INFO ] : 解析$.token得到的结果是417475e64ca64b0e9c5d16c94d0c9000存储为token
20:53:32  [INFO ] : {"status":401,"msg":"用户名密码错误"}
20:53:32  [ERROR] : 测试失败，实际值是用户名密码错误
20:53:32  [INFO ] : {"status":201,"msg":"success"}
20:53:32  [ERROR] : 解析json内容失败
java.lang.NullPointerException: null
	at com.testing.inter.InterKeyword.saveJsonParam(InterKeyword.java:202) [classes/:?]
	at com.testing.class18.LoginAPI.testLoginParam(LoginAPI.java:43) [test-classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_251]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_251]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_251]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_251]
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124) [testng-6.14.3.jar:?]
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583) [testng-6.14.3.jar:?]
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719) [testng-6.14.3.jar:?]
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989) [testng-6.14.3.jar:?]
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125) [testng-6.14.3.jar:?]
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109) [testng-6.14.3.jar:?]
	at org.testng.TestRunner.privateRun(TestRunner.java:648) [testng-6.14.3.jar:?]
	at org.testng.TestRunner.run(TestRunner.java:505) [testng-6.14.3.jar:?]
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455) [testng-6.14.3.jar:?]
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450) [testng-6.14.3.jar:?]
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415) [testng-6.14.3.jar:?]
	at org.testng.SuiteRunner.run(SuiteRunner.java:364) [testng-6.14.3.jar:?]
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52) [testng-6.14.3.jar:?]
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84) [testng-6.14.3.jar:?]
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208) [testng-6.14.3.jar:?]
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137) [testng-6.14.3.jar:?]
	at org.testng.TestNG.runSuites(TestNG.java:1049) [testng-6.14.3.jar:?]
	at org.testng.TestNG.run(TestNG.java:1017) [testng-6.14.3.jar:?]
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66) [testng-rt.jar:?]
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110) [testng-rt.jar:?]
20:53:32  [INFO ] : {"status":401,"msg":"用户名密码错误"}
20:53:32  [ERROR] : 测试失败，实际值是用户名密码错误
